Resources/Locale/en-US/accent/italian.ftl

```diff
@@ -1,10 +1,10 @@
 # This should probably use the same prefix system as the mobster accent.
 # For the record, these do not work right now - even when uncommented.
 
-# accent-italian-prefix-1 = Ravioli, ravioli, give me the formuoli!
-# accent-italian-prefix-2 = Mamma-mia!
-# accent-italian-prefix-3 = Mamma-mia! That's a spicy meat-ball!
-# accemt-italian-prefix-4 = La la la la la funiculi funicula!
+accent-italian-prefix-1 = Ravioli, ravioli, give me the formuoli!
+accent-italian-prefix-2 = Mamma-mia!
+accent-italian-prefix-3 = Mamma-mia! That's a spicy meat-ball!
+accent-italian-prefix-4 = La la la la la funiculi funicula!
```

Comment:
These are quite long and will likely be perceived by many players as annoying. Please shorten or remove these.